flow/layers/clip_path_layer.cc

```diff
@@ -17,8 +22,25 @@ void ClipPathLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   set_paint_bounds(context->child_paint_bounds);
 }
 
+#if defined(OS_FUCHSIA)
+
+void ClipPathLayer::UpdateScene(SceneUpdateContext& context,
```

Comment:
There is a lot of code repetition here for only needing to have specialized logic for updating the node. Maybe we can move the bulk of this logic into the base and have a virtual method that takes a node and updates it appropriately (`prepareNode` or something similar) per subclass.